Tools/environment_install/install-ROS-ubuntu.sh

```diff
@@ -209,8 +209,43 @@ heading "${green}Initializaing rosdep${reset}"
 sudo rosdep init || true
 # To find available packages, use:
 rosdep update
+
+# Installing ardupilot_gazebo
+git clone https://github.com/SwiftGust/ardupilot_gazebo
```

Comment:
this should point to https://github.com/ardupilot/ardupilot_gazebo
If that repo is missing something when we should make pull requests to that repo to bring it up to date.

Reply:
Actually this is incorrect... this should be cloned into the workspace and be part of the ardupilot_ws .....